src/prettytable/prettytable.py

```diff
@@ -2214,10 +2250,11 @@ def _get_simple_html_string(self, options):
             for field, datum in zip(self._field_names, row):
                 if options["fields"] and field not in options["fields"]:
                     continue
+                if options["escape_data"]:
+                    datum = escape(datum)
```

Comment:
Tests still pass without this.

Reply:
What do you mean?
If I remove that, the tests fails for me
![image](https://github.com/user-attachments/assets/fd71daf5-1889-43b7-9a40-9757a113c172)
![image](https://github.com/user-attachments/assets/e5d5659a-1c41-4a18-b817-9b9a1b56fc8b)

Reply:
Sorry, I meant line 2344.